src/lexer.coffee

```diff
@@ -369,18 +369,31 @@ exports.Lexer = class Lexer
           comment
         .filter (comment) -> comment
 
+    getIndentSize = ({leadingWhitespace, nonInitial}) ->
+      lastNewlineIndex = leadingWhitespace.lastIndexOf '\n'
```

Comment:
We normalize line endings somewhere long before this, right? So we don’t need to worry about `\r` or the other variations?

Reply:
Yes `\r` has already been removed

We will however need to account for those normalizations in order to maintain accurate location data (I believe that there are ESLint plugin tests that will exercise that so will include it in the next AST PR, it's definitely something I added coverage for in my `prettier` branch)